### PR TITLE
update description

### PR DESCRIPTION
**Pull request by elenapeskova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,24 +1,113 @@
...
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [DataProvider.cs](./CS/WebSite/App_Code/DataProvider.cs) (VB: [DataProvider.vb](./VB/WebSite/App_Code/DataProvider.vb))
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to implement cascaded combo box columns in ASPxGridView without using templates
+# ASPxGridView for ASP.NET Web Forms - Cascading Combo Boxes
```
Comment (LanaDX) [inline Readme.md]:
```suggestion
# Grid View for ASP.NET Web Forms - Cascading Combo Boxes
```

File: Readme.md [hunk 2]
```diff
@@ -1,24 +1,113 @@
...
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e3689/)**
 <!-- run online end -->
 
+This example demonstrates how to implement cascading combo box editors within [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView).
+
+![CascadingComboBoxes](~/images/CascadingComboBoxes.png)
```
Comment (LanaDX) [inline Readme.md]:
```suggestion
![CascadingComboBoxes](images/CascadingComboBoxes.png)
```

File: Readme.md [hunk 3]
```diff
@@ -1,24 +1,113 @@
...
+    <SettingsEditing Mode="Inline" />
+</dx:ASPxGridView>
+<asp:ObjectDataSource ID="Customers" runat="server" ... />
+<asp:ObjectDataSource ID="Countries" runat="server" ... />
+<asp:ObjectDataSource ID="AllCities" runat="server" ... />
+```
+
+## Respond to a Selection Change on the Client Side
+
+Handle the primary editor's client-side [SelectedIndexChanged](https://docs.devexpress.com/AspNet/js-ASPxClientComboBox.SelectedIndexChanged) event. In this event handler, get the current editor's value (the [GetValue](https://docs.devexpress.com/AspNet/js-ASPxClientEditBase.GetValue) method) and pass it as a parameter in the [PerformCallback](https://docs.devexpress.com/AspNet/js-ASPxClientComboBox.PerformCallback(parameter)) method of the secondary editor. To access the secondary editor, call the [GetEditor](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.GetEditor(column)) method.
```
Comment (LanaDX) [inline Readme.md]: current editor's value - its a value of the current editor. I think, you mean current value of editor. Anyway, I would remove the `current` at all
```suggestion
Handle the primary editor's client-side [SelectedIndexChanged](https://docs.devexpress.com/AspNet/js-ASPxClientComboBox.SelectedIndexChanged) event. In this event handler, get the editor value (the [GetValue](https://docs.devexpress.com/AspNet/js-ASPxClientEditBase.GetValue) method) and pass it as a parameter in the [PerformCallback](https://docs.devexpress.com/AspNet/js-ASPxClientComboBox.PerformCallback(parameter)) method of the secondary editor. To access the secondary editor, call the [GetEditor](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.GetEditor(column)) method.
```

File: Readme.md [hunk 4]
```diff
@@ -1,24 +1,113 @@
...
+```js
+function CountriesCombo_SelectedIndexChanged(s, e) {
+    grid.GetEditor("CityID").PerformCallback(s.GetValue());
+}
+```
 
-<br/>
+## Filter the Secondary Combo Box Values on the Server Side
 
+In the grid's [CellEditorInitialize](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.CellEditorInitialize) event handler, access the secondary editor and handle its [Callback](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxCallback.Callback) event. In this event handler, use the [Parameter](https://docs.devexpress.com/AspNet/DevExpress.Web.CallbackEventArgsBase.Parameter) argument property to obtain the primary editor's value from the client side. Filter the secondary editor's data source based on this value.
```
Comment (LanaDX) [inline Readme.md]:
```suggestion
In the [CellEditorInitialize](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.CellEditorInitialize) event handler, access the secondary editor and add a handler to its [Callback](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxCallback.Callback) event. In the handler, use the [Parameter](https://docs.devexpress.com/AspNet/DevExpress.Web.CallbackEventArgsBase.Parameter) argument property to obtain the primary editor's value from the client side. Filter the secondary editor's data source based on this value and bind the filtered values to the editor.
```

File: Readme.md [hunk 5]
```diff
@@ -1,24 +1,113 @@
...
 **[[Run Online]](https://codecentral.devexpress.com/e3689/)**
 <!-- run online end -->
 
+This example demonstrates how to implement cascading combo box editors within [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView).
+
+![CascadingComboBoxes](images/CascadingComboBoxes.png)
+
+## Setup the Grid and its Column Editors
+
+Create an [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) control, assign its data source, and set the grid's [edit mode](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridViewEditingSettings.Mode) to `Inline`. Add two columns of the [GridViewDataComboBoxColumn](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewDataComboBoxColumn) type and set their data sources.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
Create an [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) control, assign its data source, and set the grid's [edit mode](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridViewEditingSettings.Mode) to `Inline`. Add two columns of the [GridViewDataComboBoxColumn](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewDataComboBoxColumn) type and specify their data sources.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**